plugins/reolink/src/main.ts

```diff
@@ -703,21 +703,30 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, DeviceProvider, R
         // anecdotally, encoders of type h265 do not have a working RTMP main stream.
         const mainEncType = this.storageSettings.values.abilities?.value?.Ability?.abilityChn?.[rtspChannel].mainEncType?.ver;
 
-        if (live === 2) {
-            if (mainEncType === 1) {
-                streams.push(rtmpSub, rtspMain, rtspSub);
+        const isNvr = this.client.getIsNvr(deviceInfo);
+
+        if (isNvr) {
```

Comment:
Interesting comparing, this is the only block added here (708-719), the non nvr branch is not changed at all